src/app/record/components/work-stack-group/work-stack-group.component.html

```diff
@@ -76,14 +76,44 @@
             <span class="action" i18n="@@shared.actions"> Actions </span>
             <mat-icon class="large-material-icon">arrow_drop_down</mat-icon>
           </button>
-          <mat-menu #menu="matMenu">
+          <mat-menu #menu="matMenu" class="menu-buttons">
             <button
               mat-menu-item
               (click)="combine()"
-              [disabled]="selectedWorks.length === 0"
+              [disabled]="
+                selectedWorks.length === 0 || selectedWorks.length > 5
+              "
             >
-              <mat-icon>merge_type</mat-icon>
-              <span i18n="@@shared.combineWorks">Combine works</span>
+              <div>
+                <mat-icon *ngIf="selectedWorks.length <= 5"
+                  >merge_type</mat-icon
+                >
+                <mat-icon class="error-icon" *ngIf="selectedWorks.length > 5"
+                  >error_outline</mat-icon
+                >
+              </div>
+              <div>
+                <div
+                  [ngClass]="{
+                    'group-selected-works-label': selectedWorks.length > 5
+                  }"
+                >
+                  <ng-container i18n="@@shared.combineWorks"
```

Comment:
CI is failing because `shared.combineWorks` is missing from `src/locale/properties/works/works.en.properties`